bench/mjolnir/config.json

```diff
@@ -0,0 +1,256 @@
+{
```

Comment:
have doubts about committing the whole config file

Reply:
Just an idea: could the config file be generated at runtime using `scripts/valhalla_build_config`?

Reply:
I think it's not a big deal to commit this file. We could generate it at build/CMake time. In the past, I've also tried leaving out parts of the configuration that I thought were unused, but the configuration reader in those cases will throw an exception or segfault.

One alternative, if we're worried about configuration sprawl (not a historical concern), would be to have a shared default, configuration that could be used by some of the benchmarks. The `meili` benchmark config file actually has nothing special about it.

Reply:
cmake-idea sounds good to me